polyphemus/lib/server.rb

```diff
@@ -1,9 +1,22 @@
+require 'json'
+require 'rack'
+
+require_relative './polyphemus'
+require_relative './polyphemus/controllers/configuration_controller'
+
 class Polyphemus
   class Server < Etna::Server
     def initialize
       super
       # application.load_models
       application.setup_db
     end
+
+    # Return the app host configuration values
+    get '/configuration', as: :configuration, action: 'configuration#action', auth: { user: { is_superuser?: true } }
```

Comment:
We can relax this if needed for third-party users? I figure all DS will need to be superusers to create projects, so this auth requirement is sufficient for now.

Reply:
True.  To be honest... I'm not sure that the configuration endpoint needs any auth.  None of it is really secret per say?  If it's just hostnames.